internal/resolvers/default/thumbnail.go

```diff
@@ -19,11 +14,10 @@ import (
 	"github.com/Chatterino/api/pkg/resolver"
 	"github.com/Chatterino/api/pkg/utils"
 	"github.com/go-chi/chi/v5"
-	"github.com/nfnt/resize"
 )
 
 var (
-	supportedThumbnails = []string{"image/jpeg", "image/png", "image/gif"}
+	supportedThumbnails = []string{"image/jpeg", "image/png", "image/gif", "image/webp"}
```

Comment:
Maybe not really related to the PR itself, but why is this a type of var? (also, vars should be under const if anything).